docs/guide/rules.md

```diff
@@ -379,6 +379,13 @@ The field under validation must be an iterable and/or have a length property of
 <input v-validate="'length:5'" type="text" name="field">
 ```
 
+Specifying both len and max:
+
+```html
+<input v-validate="'length:[2, 3]'" type="checkbox" name="field">
+```
+The above example will make sure at least 2 checkboxes are selected and at most 3 are selected.
```

Comment:
Since the `required` rule is missing, it will do as you described but if it's not empty. Which means leaving the array empty will pass the validation. Maybe this should be also mentioned here.

Reply:
I just tested it, it doesn't work as you described. Leaving it empty fails validation when using only length.
I think it will be a bug if it __didnt__. Saying you want the length to be between 2 and 3 should make length of 0 fail validation. And it seems to work as I will expect.

[https://codesandbox.io/s/18qp7m0r4l](https://codesandbox.io/s/18qp7m0r4l)

Reply:
This is actually a bug, the empty values should include an empty array. Thanks for bringing this up, this actually an edge case since the empty array would pass the required rule which it shouldn't.

Reply:
This is fixed in aed1c5eb5ee5a391126a2e3bc6e77ed3287631a3 If you would like to update the PR to reflect this that would be great.